parser/ndt.go

```diff
@@ -177,11 +181,10 @@ func (n *NDTParser) ParseAndInsert(taskInfo map[string]bigquery.Value, testName
 					n.TableName(), "c2s", "timestamp collision").Inc()
 				log.Printf("Collision: %s and %s\n", n.c2s.fn, testName)
 			}
-		} else {
-			// We only take action on the first file, to avoid dups.
-			n.c2s = &fileInfoAndData{testName, *info, content}
-			n.processTest(taskFileName, n.c2s, "c2s")
 		}
+		// We always use the latest file, since .gz is more reliably
+		// complete, and lexicographically later.
```

Comment:
Okay, but then this ups the priority on `TODO(prod): make sure historical tar files are also in lexicographic order`